report.js

```diff
@@ -70,9 +70,11 @@ let stats = {
 
 Promise.all([
   fetch("report.json").then(r => r.json()),
-  fetch("annotations.json").then(r => r.json())
+  fetch("annotations.json").then(r => r.json()),
+  fetch("https://api.w3.org/groups/cg?items=500&embed=1").then(r => r.json()).then(data => data._embedded.groups.filter(g => !g['spec-publisher']).map(g => g.shortname)),
```

Comment:
Rather than fetching the information from the API when displaying the report, I think that data should be integrated in the aggregated data report collected through `monitor.js`. This would be a fairly trivial change, except that `https://api.w3.org/groups?embed=1` doesn't include the `spec-publisher` property. Is that on purpose or difficult to fix?

Reply:
Thanks for catching this. There was indeed a cache issue. `spec-publisher` should now be listed properly.
I agree fetching data like that will be sub-optimal. I'll see how to update monitor.js so don't have to fetch the data each time.

Reply:
I also added the property only for CG and BG but we can probably extend this to the other types of groups

Reply:
addressed by https://github.com/w3c/cg-monitor/pull/37/commits/8d76c72112964e373c8325b0d65e5622ca15e586